packages/apollo-cli/src/commands/assembly/add-fasta.ts

```diff
@@ -43,25 +36,32 @@ export default class Get extends BaseCommand<typeof Get> {
     }),
     index: Flags.string({
       char: 'x',
-      description:
-        'URL of the index. Required if input is an external source and ignored if input is a local file',
+      description: 'URL of the index. Required if input is an external source',
     }),
     force: Flags.boolean({
       char: 'f',
       description: 'Delete existing assembly, if it exists',
     }),
+    'no-db': Flags.boolean({
```

Comment:
`no-db` should be something like `not-editable`, since that what is relevant to the user (the database is an internal detail they don't need to worry about)